flag_impl.go

```diff
@@ -192,7 +192,8 @@ func (f *FlagBase[T, C, V]) RunAction(ctx *Context) error {
 // IsSliceFlag returns true if the value type T is of kind slice
 func (f *FlagBase[T, C, VC]) IsSliceFlag() bool {
```

Comment:
I think we need to rename this to IsMultiValueFlag as we are overloading this function name now . I can make that change in a separate PR.

Reply:
@avorima  PR #1615